scripts/shared/lib/utils

```diff
@@ -105,12 +101,33 @@ function declare_cidrs() {
     # shellcheck disable=SC2034 # these variables are used elsewhere
     declare -gA cluster_CIDRs service_CIDRs global_CIDRs
 
-    for i in {1..3}; do
-        add_cluster_cidrs $i
+    i=1
```

Comment:
You should notice this function is also called here: https://github.com/submariner-io/submariner-operator/blob/master/scripts/kind-e2e/lib_operator_verify_subm.sh#L27

I think it's safer is you just handle clusters internally

Reply:
That's fine - either way submariner will need to change to source _cluster_settings_.